src/app/dao/tests/FirestoreDAOMock.spec.ts

```diff
@@ -113,7 +113,7 @@ export abstract class FirestoreDAOMock<T extends FirestoreJSONObject> implements
             const subject: BehaviorSubject<MGPOptional<FirestoreDocument<T>>> =
                 new BehaviorSubject(MGPOptional.of(tid));
             const observable: Observable<MGPOptional<FirestoreDocument<T>>> = subject.asObservable();
-            this.getStaticDB().put(id, new ObservableSubject(subject, observable));
+            this.getStaticDB().set(id, new ObservableSubject(subject, observable));
```

Comment:
une raison particulière de remplacer le put par un set ?

Reply:
un refactor gone wrong !
PLEASE REPORT ALL OF THOSES !

Reply:
put put put, sale-set, sale-set, sale-set! J'essaye de tous les trouver alors !

Reply:
put put put, sale-set, sale-set,sale-set ??? :D ? QUEUWA ?

Reply:
euuuh je sais pas c'est une réf à quoi, mais c'est bien possible :D 
Y'a plus de .put dans le diff, tu peux donc résoudre !